app/views/accounts/_backup_codes.html.erb

```diff
@@ -2,13 +2,6 @@
   <h2 class="grid-col-fill margin-y-0 padding-right-2">
```

Comment:
Same note about removing grid here. I think this can be:

```suggestion
  <h2 class="margin-top-0 margin-bottom-1">
```

And remove the parent wrapper altogether.